CalibMuon/DTCalibration/plugins/DTTTrigCorrectionFirst.cc

```diff
@@ -86,7 +85,6 @@ void DTTTrigCorrectionFirst::endJob() {
     tTrigMap->get((*sl)->id(), ttrigMean, ttrigSigma, kfactor, DTTimeUnits::ns);
     if (ttrigMean < ttrigMax && ttrigMean > ttrigMin) {
       average2 += (ttrigMean - average) * (ttrigMean - average);
-      average2Sigma += (ttrigSigma - averageSigma) * (ttrigSigma - averageSigma);
```

Comment:
@namapane is this supposed to be used for something?

Reply:
@tvami , this was used in a `cout` statement but https://github.com/cms-sw/cmssw/pull/36534 has removed its usage

Reply:
indeed, should be safe to remove.